kmd.tree

```diff
@@ -22,9 +22,12 @@
         <toc-element id="organize-process-around-kmm.md"/>
         <toc-element id="architect-kmm-app.md"/>
         <toc-element id="connect-to-platform-specific-apis.md"/>
-        <toc-element id="configure-sqldelight-for-data-storage.md"/>
-        <toc-element id="use-ktor-for-networking.md"/>
         <toc-element id="add-dependencies.md"/>
+        <toc-element toc-title="Networking and data storage">
```

Comment:
`Networking and data storage` is not consistent with other titles of the same level. Please add a verb.

I would put a hands-on tutorial as the first item since it covers basics. Anyway, I would review the new structure in the channel with Nastya and Kate P.